### PR TITLE
Test the gossipsub message size limit

### LINKED ISSUE #2529: 1mb max message size is too small for macroblocks
Network libp2p. Config. 1mb max message size for gossip sub can easily be exceeded for macro blocks due to reward txs.

https://github.com/nimiq/core-rs-albatross/blob/1cf1c3f4e503d199d5ff0c123d13660b23215d3c/network-libp2p/src/config.rs#L57

### PR DESCRIPTION
**Pull request by hrxi**
Make sure it's well above the size of a macro block.

Fixes #2529.

### CODE REVIEW

**Comment by hrxi** [comment]
The current maximum size of a macro block is 201_832 bytes, so roughly a fifth of the current maximum message size.

File: network-interface/src/network.rs [hunk 1]
```diff
@@ -37,6 +37,8 @@ pub trait Topic {
     const VALIDATE: bool;
 }
 
+pub const MIN_SUPPORTED_MSG_SIZE: usize = 1024 * 1024;
```
Comment (sisou) [inline network-interface/src/network.rs:41]: Do you mean `MAX`?

Comment (hrxi) [reply network-interface/src/network.rs:41]: I had a hard time naming this constant.

I want to say that this is the minimum maximum message size of the underlying network. I.e. the network has to at least support messages of this size. I don't want to tell the underlying network that it has to limit the sizes to exactly this, just that it needs to support messages of this size.

Comment (sisou) [reply network-interface/src/network.rs:41]: That's great. I'd suggest putting a rustdoc comment on it, explaining it exactly like that:

```rust
/// Network implementations have to at least support messages of this size
pub const MIN_SUPPORTED_MSG_SIZE: usize = 1024 * 1024;
```

Comment (hrxi) [reply network-interface/src/network.rs:41]: Thanks for the concise description. Added.

File: primitives/block/src/multisig.rs [hunk 2]
```diff
@@ -15,6 +16,10 @@ pub struct MultiSignature {
 }
 
 impl MultiSignature {
```
Comment (jsdanielh) [inline primitives/block/src/multisig.rs:18]:
```suggestion
impl MultiSignature {
 #[allow(clippy::identity_op)]
```

Comment (hrxi) [reply primitives/block/src/multisig.rs:18]: Done.

**Review by jsdanielh — APPROVED** [review]

**Pull request merged**